server/lib/src/idm/ldap.rs

```diff
@@ -477,6 +478,94 @@ impl LdapServer {
         Ok(result)
     }
 
+    #[instrument(level = "debug", skip_all)]
+    async fn do_compare(
+        &self,
+        _idms: &IdmServer,
+        cr: &CompareRequest,
+        uat: &LdapBoundToken,
+        _source: Source,
+    ) -> Result<Vec<LdapMsg>, OperationError> {
+        admin_info!("Attempt LDAP CompareRequest for {}", uat.spn);
```

Comment:
this should probably be a trace/debug level message

Reply:
I can change it of course - but I just replicated the search method: https://github.com/kanidm/kanidm/pull/2780/files/a3e46270e7cba16242619c10505983be01b9eee0#diff-3a38c3a25731778180d0864ffe0f2987822f93891340f37a61d05d4207afc1e7R148

What do you prefer for me to do?

Reply:
m'bad, I went looking for a similar message and didn't find it (was on my phone 😇) leave it as-is, thanks.